### PR TITLE
verification codes and docs for immuno tables

### PR DESCRIPTION
**Pull request by yiwenlu**

### CODE REVIEW

File: immuno_tabular/verification/immuno_table_verification.Rmd [hunk 1]
```diff
@@ -0,0 +1,1193 @@
...
+
+rr_table2_all <- do.call("rbind", rr_table2_list)
+dim(rr_table2_all)==dim(tab_bind)
+
+tab_bind[] <- lapply(tab_bind, as.character)
+rr_table2_all[] <- lapply(rr_table2_all, as.character)
+
+rr_table2_all = tab_bind[,c("subgroup","Group","Visit","Arm","Baseline SARS-CoV-2","Marker")] %>% left_join(rr_table2_all, by=c("subgroup","Group","Visit","Arm","Baseline SARS-CoV-2","Marker"))
+
+testthat::expect_equal(rr_table2_all, tab_bind)
```
Comment (thebioengineer) [inline immuno_tabular/verification/immuno_table_verification.Rmd:360]: Is this to make sure the order matches?

**Review by thebioengineer — CHANGES_REQUESTED** [review]
thanks for the PR @yiwenlu. Looks great overall. It looks like you are doing all your processing and then comparison within the same Rmd. 

We would need to update the docs we have within the `verification` folder to point to the output file from this Rmd, and add some text in there indicating you've done a complete independent double programming and have compared the results and they match. see [this](https://github.com/CoVPN/correlates_reporting/blob/master/data_clean/verification/verification_report.Rmd) for an example of the code and [this]( https://github.com/CoVPN/correlates_reporting/blob/master/data_clean/verification/verification_report.pdf) for the PDF output we might want

File: immuno_tabular/verification/immuno_table_verification.Rmd (hunk 1 above)
Comment (yiwenlu) [reply immuno_tabular/verification/immuno_table_verification.Rmd:360]: Yes, these are for sorting orders

Comment (yiwenlu) [reply immuno_tabular/verification/immuno_table_verification.Rmd:360]: @thebioengineer Does it need to be in csv format? All the files I received from Chenchen is in csv format though...

Comment (thebioengineer) [reply immuno_tabular/verification/immuno_table_verification.Rmd:360]: It looks like Chenchen bundled all the tables together for you in the RDA - make sure to comment that in your report and I think it should be okay. I would prefer csv's so when one table changes Chenchen only has to send you the one updated table, but its fine.

**Pull request closed without merging**